pkg/bpftrace/build.yml

```diff
@@ -0,0 +1,6 @@
+# linuxkit build template
+#
+# SPDX-License-Identifier: Apache-2.0
+org: lfedge
+image: eve-bpftrace
+network: yes
```

Comment:
Can you turn off the network?
We build as much as possible that way to ensure that any source is done using ADD commands so that they make it into the SPDX SBoM file. 
If networking is enabled then someone might later add some curl/wget/git pull way to add source which would not be included in the SBoM.